aio_pika/robust_connection.py

```diff
@@ -64,6 +74,10 @@ def _on_connection_close(self, connection, closing, *args, **kwargs):
 
         super()._on_connection_close(connection, closing)
 
+        log.info(
+            "Connection to %s closed. Reconnecting after %r seconds.",
```

Comment:
after **%d** seconds

Reply:
`%r` returns same result but without problem when non integer passed.

Reply:
It's a little bit confusing that %s, %d and %r are mixed. I mean sometimes it is %d, but sometimes it is %r and so on..

Reply:
My rule is "Type `%r` when you log user-defined parameters"

Reply:
I mean usage of "after %d seconds" below (line 137) but "after %r seconds" here for example. Anyway, let's merge it.

Reply:
Fine.